File: python/7 kyu/327.py
# https://www.codewars.com/kata/5e030f77cec18900322c535d/train/python

# Passed

from math import ceil

def minimum(a, x):
    result = min(a % x, x * ceil(a/x) - a)
    return result

output = minimum(10, 6)
print(output)

File: nodejs/7 kyu/327.js
// https://www.codewars.com/kata/5806c2f897dba05dd900004c/train/javascript

// Passed

function magnitude(vector) {
  let squareSum = vector.map((val) => val ** 2).reduce((a, b) => a + b, 0);
  let result = squareSum ** 0.5;
  return result;
}

const output = magnitude([-2, -4, 4]);
console.log(output);